src/pombot/commands/help.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * A command to show help information.
+ */
+import {createCommand} from 'chatter';
+
+export default createCommand({
```

Comment:
If you define a `help` command, the built-in one will never run. But you want the built-in one to run, because it does all the hard work for you! As long as you give commands a `name` and `description` and optional `usage`, the built-in `help` will do its thing.